corehq/motech/generic_inbound/utils.py

```diff
@@ -181,12 +181,13 @@ def revert_api_request_from_form(form_id):
         return
 
 
-def log_api_request(api, request, response):
+def log_api_request(request_id, api, request, response):
     if _request_too_large(request):
         body = '<truncated>'
     else:
         body = as_text(request.body)
     log = RequestLog.objects.create(
+        id=request_id,
```

Comment:
Handy that this is a UUID, seems nice to have a linkage even outside of the HL7 context